src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -371,14 +431,18 @@ private DocumentedFeature getDocumentedFeatureForClass(final Class<?> clazz) {
      * @param groupMaps
      * @throws IOException
      */
-    private void processIndexTemplate(
+    protected void processIndexTemplate(
```

Comment:
I wanted also this to be open...can you separate this in a different PR to get it in faster? I would like to be able to process several indexes, and this is the only way of doing it (I guess).